app/src/main/java/com/chattymin/sopt_compose/feature/main/MainPage.kt

```diff
@@ -0,0 +1,88 @@
+package com.chattymin.sopt_compose.feature.main
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavController
+import com.chattymin.sopt_compose.R
+import com.chattymin.sopt_compose.components.spacer.Spacer
+import org.orbitmvi.orbit.compose.collectAsState
+
+@Composable
+fun MainPage(navController: NavController) {
+    val viewModel: MainViewModel = viewModel()
+
+    val state by viewModel.collectAsState()
```

Comment:
```suggestion
    val state by viewModel.collectAsStateWithLifecycle()
```

공식문서에서 권장하는 [방식](https://developer.android.com/jetpack/compose/state#use-other-types-of-state-in-jetpack-compose)은 위와 같습니다.

Reply:
이 부분은 orbit-mvi를 사용하고 있어서 다른 것 같습니다!
`collectAsStateWithLivecycle()`과 마찬가지로 orbit의 `collectAsState()`는 `Lifecycle.State.STARTED`로 라이프사이클을 연결해주고 있습니다!!

orbit-mvi를 사용하지 않는 환경으로 개발을 진행하게 되면 유의해서 코드 작성하도록 하겠습니다 :)
```
@Composable 
fun <STATE : Any, SIDE_EFFECT : Any> ContainerHost<STATE, SIDE_EFFECT>.collectAsState( 
    lifecycleState: Lifecycle.State = Lifecycle.State.STARTED 
): State<STATE>

```